template/src/lib.rs

```diff
@@ -16,7 +16,7 @@ static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
 pub fn main_js() -> Result<(), JsValue> {
     // This provides better error messages in debug mode.
     // It's disabled in release mode so it doesn't bloat up the file size.
```

Comment:
This should also be updated:

```suggestion
    // It should be disabled in release mode so it doesn't bloat up the file size.
```